src/main/java/uk/ac/ebi/ena/webin/cli/manifest/processor/metadata/StudyProcessor.java

```diff
@@ -57,7 +58,7 @@ public void process(ValidationResult result, ManifestFieldValue fieldValue) {
       callback.notify(study);
 
     } catch (WebinCliException e) {
-      if (WebinCliMessage.CLI_AUTHENTICATION_ERROR.text().equals(e.getMessage())) {
+      if (WebinCliMessage.SERVICE_AUTHENTICATION_ERROR.format(StudyService.SERVICE_NAME).equals(e.getMessage())) {
```

Comment:
Simplify to:

} catch (WebinCliException e) {
    throw e;
}
catch (Exception e) {
   throw WebinCliException.systemError(e, WebinCliMessage.STUDY_SERVICE_SYSTEM_ERROR.format(sampleId));
}